ci/nfpm/template.yaml

```diff
@@ -124,7 +124,7 @@ contents:
       mode: 0744
 
   - src: ${WORKDIR}/bin/deps/lib/${ARCH}/latest
-    dst: /usr/lib/
+    dst: /usr/lib/${NAME}
```

Comment:
I don't know how its on Red Hat, but on Debian-based distributions shared libraries should be placed within architecture directory, for example `/usr/lib/x86_64-linux-gnu/nordvpn`.

Reply:
Yes, it's a bit of a mess. Usually shared libs should be put in `/usr/lib64/..` or as you mentioned in `/usr/lib/<arch>/..` but this is for system-wide libs. `libtelio.so` etc are not considered system wide and also we currently don't have a mechanism to detect the arch in scriptlets to properly refresh linker cache, so I ended up putting it in `/usr/lib/nordvpn`. I'll create a ticket to improve this